pkgs/data/fonts/victor-mono/default.nix

```diff
@@ -17,11 +18,11 @@ fetchzip {
 
   postFetch = ''
     mkdir -p $out/share/fonts/
-    unzip -j $downloadedFile \*.otf -d $out/share/fonts/opentype
-    unzip -j $downloadedFile \*.ttf -d $out/share/fonts/truetype
+    cp --recursive $out/OTF $out/share/fonts/opentype
```

Comment:
I'm not sure exactly why, but the resulting tree under nixpkgs is much bigger after the changes for 1.5.3 in this MR:

Are the directories outside of `/share/fonts/` really needed?

```
/nix/store/4qj2nbjzrr7nhf23gg9fcp6rb29sl93f-victor-mono-1.5.2
└── share
    └── fonts
        ├── opentype
        │   ├── VictorMono-BoldItalic.otf
        │   ├── VictorMono-BoldOblique.otf
        │   ├── VictorMono-Bold.otf
        │   ├── VictorMono-ExtraLightItalic.otf
        │   ├── VictorMono-ExtraLightOblique.otf
        │   ├── VictorMono-ExtraLight.otf
        │   ├── VictorMono-Italic.otf
        │   ├── VictorMono-LightItalic.otf
        │   ├── VictorMono-LightOblique.otf
        │   ├── VictorMono-Light.otf
        │   ├── VictorMono-MediumItalic.otf
        │   ├── VictorMono-MediumOblique.otf
        │   ├── VictorMono-Medium.otf
        │   ├── VictorMono-Oblique.otf
        │   ├── VictorMono-Regular.otf
        │   ├── VictorMono-SemiBoldItalic.otf
        │   ├── VictorMono-SemiBoldOblique.otf
        │   ├── VictorMono-SemiBold.otf
        │   ├── VictorMono-ThinItalic.otf
        │   ├── VictorMono-ThinOblique.otf
        │   └── VictorMono-Thin.otf
        └── truetype
            ├── VictorMono-BoldItalic.ttf
            ├── VictorMono-BoldOblique.ttf
            ├── VictorMono-Bold.ttf
            ├── VictorMono-ExtraLightItalic.ttf
            ├── VictorMono-ExtraLightOblique.ttf
            ├── VictorMono-ExtraLight.ttf
            ├── VictorMono-Italic.ttf
            ├── VictorMono-LightItalic.ttf
            ├── VictorMono-LightOblique.ttf
            ├── VictorMono-Light.ttf
            ├── VictorMono-MediumItalic.ttf
            ├── VictorMono-MediumOblique.ttf
            ├── VictorMono-Medium.ttf
            ├── VictorMono-Oblique.ttf
            ├── VictorMono-Regular.ttf
            ├── VictorMono-SemiBoldItalic.ttf
            ├── VictorMono-SemiBoldOblique.ttf
            ├── VictorMono-SemiBold.ttf
            ├── VictorMono-ThinItalic.ttf
            ├── VictorMono-ThinOblique.ttf
            └── VictorMono-Thin.ttf

4 directories, 42 files
```

vs

```
/nix/store/fcdivhrjbv8qbpd9w99ks9is1nd16hww-victor-mono-1.5.3
├── EOT
│   ├── VictorMono-Bold.eot
│   ├── VictorMono-BoldItalic.eot
│   ├── VictorMono-BoldOblique.eot
│   ├── VictorMono-ExtraLight.eot
│   ├── VictorMono-ExtraLightItalic.eot
│   ├── VictorMono-ExtraLightOblique.eot
│   ├── VictorMono-Italic.eot
│   ├── VictorMono-Light.eot
│   ├── VictorMono-LightItalic.eot
│   ├── VictorMono-LightOblique.eot
│   ├── VictorMono-Medium.eot
│   ├── VictorMono-MediumItalic.eot
│   ├── VictorMono-MediumOblique.eot
│   ├── VictorMono-Oblique.eot
│   ├── VictorMono-Regular.eot
│   ├── VictorMono-SemiBold.eot
│   ├── VictorMono-SemiBoldItalic.eot
│   ├── VictorMono-SemiBoldOblique.eot
│   ├── VictorMono-Thin.eot
│   ├── VictorMono-ThinItalic.eot
│   └── VictorMono-ThinOblique.eot
├── LICENSE.txt
├── OTF
│   ├── VictorMono-BoldItalic.otf
│   ├── VictorMono-BoldOblique.otf
│   ├── VictorMono-Bold.otf
│   ├── VictorMono-ExtraLightItalic.otf
│   ├── VictorMono-ExtraLightOblique.otf
│   ├── VictorMono-ExtraLight.otf
│   ├── VictorMono-Italic.otf
│   ├── VictorMono-LightItalic.otf
│   ├── VictorMono-LightOblique.otf
│   ├── VictorMono-Light.otf
│   ├── VictorMono-MediumItalic.otf
│   ├── VictorMono-MediumOblique.otf
│   ├── VictorMono-Medium.otf
│   ├── VictorMono-Oblique.otf
│   ├── VictorMono-Regular.otf
│   ├── VictorMono-SemiBoldItalic.otf
│   ├── VictorMono-SemiBoldOblique.otf
│   ├── VictorMono-SemiBold.otf
│   ├── VictorMono-ThinItalic.otf
│   ├── VictorMono-ThinOblique.otf
│   └── VictorMono-Thin.otf
├── share
│   └── fonts
│       ├── opentype
│       │   ├── VictorMono-BoldItalic.otf
│       │   ├── VictorMono-BoldOblique.otf
│       │   ├── VictorMono-Bold.otf
│       │   ├── VictorMono-ExtraLightItalic.otf
│       │   ├── VictorMono-ExtraLightOblique.otf
│       │   ├── VictorMono-ExtraLight.otf
│       │   ├── VictorMono-Italic.otf
│       │   ├── VictorMono-LightItalic.otf
│       │   ├── VictorMono-LightOblique.otf
│       │   ├── VictorMono-Light.otf
│       │   ├── VictorMono-MediumItalic.otf
│       │   ├── VictorMono-MediumOblique.otf
│       │   ├── VictorMono-Medium.otf
│       │   ├── VictorMono-Oblique.otf
│       │   ├── VictorMono-Regular.otf
│       │   ├── VictorMono-SemiBoldItalic.otf
│       │   ├── VictorMono-SemiBoldOblique.otf
│       │   ├── VictorMono-SemiBold.otf
│       │   ├── VictorMono-ThinItalic.otf
│       │   ├── VictorMono-ThinOblique.otf
│       │   └── VictorMono-Thin.otf
│       └── truetype
│           ├── VictorMono-BoldItalic.ttf
│           ├── VictorMono-BoldOblique.ttf
│           ├── VictorMono-Bold.ttf
│           ├── VictorMono-ExtraLightItalic.ttf
│           ├── VictorMono-ExtraLightOblique.ttf
│           ├── VictorMono-ExtraLight.ttf
│           ├── VictorMono-Italic.ttf
│           ├── VictorMono-LightItalic.ttf
│           ├── VictorMono-LightOblique.ttf
│           ├── VictorMono-Light.ttf
│           ├── VictorMono-MediumItalic.ttf
│           ├── VictorMono-MediumOblique.ttf
│           ├── VictorMono-Medium.ttf
│           ├── VictorMono-Oblique.ttf
│           ├── VictorMono-Regular.ttf
│           ├── VictorMono-SemiBoldItalic.ttf
│           ├── VictorMono-SemiBoldOblique.ttf
│           ├── VictorMono-SemiBold.ttf
│           ├── VictorMono-ThinItalic.ttf
│           ├── VictorMono-ThinOblique.ttf
│           └── VictorMono-Thin.ttf
├── TTF
│   ├── VictorMono-BoldItalic.ttf
│   ├── VictorMono-BoldOblique.ttf
│   ├── VictorMono-Bold.ttf
│   ├── VictorMono-ExtraLightItalic.ttf
│   ├── VictorMono-ExtraLightOblique.ttf
│   ├── VictorMono-ExtraLight.ttf
│   ├── VictorMono-Italic.ttf
│   ├── VictorMono-LightItalic.ttf
│   ├── VictorMono-LightOblique.ttf
│   ├── VictorMono-Light.ttf
│   ├── VictorMono-MediumItalic.ttf
│   ├── VictorMono-MediumOblique.ttf
│   ├── VictorMono-Medium.ttf
│   ├── VictorMono-Oblique.ttf
│   ├── VictorMono-Regular.ttf
│   ├── VictorMono-SemiBoldItalic.ttf
│   ├── VictorMono-SemiBoldOblique.ttf
│   ├── VictorMono-SemiBold.ttf
│   ├── VictorMono-ThinItalic.ttf
│   ├── VictorMono-ThinOblique.ttf
│   └── VictorMono-Thin.ttf
├── WOFF
│   ├── VictorMono-BoldItalic.woff
│   ├── VictorMono-BoldOblique.woff
│   ├── VictorMono-Bold.woff
│   ├── VictorMono-ExtraLightItalic.woff
│   ├── VictorMono-ExtraLightOblique.woff
│   ├── VictorMono-ExtraLight.woff
│   ├── VictorMono-Italic.woff
│   ├── VictorMono-LightItalic.woff
│   ├── VictorMono-LightOblique.woff
│   ├── VictorMono-Light.woff
│   ├── VictorMono-MediumItalic.woff
│   ├── VictorMono-MediumOblique.woff
│   ├── VictorMono-Medium.woff
│   ├── VictorMono-Oblique.woff
│   ├── VictorMono-Regular.woff
│   ├── VictorMono-SemiBoldItalic.woff
│   ├── VictorMono-SemiBoldOblique.woff
│   ├── VictorMono-SemiBold.woff
│   ├── VictorMono-ThinItalic.woff
│   ├── VictorMono-ThinOblique.woff
│   └── VictorMono-Thin.woff
└── WOFF2
    ├── VictorMono-BoldItalic.woff2
    ├── VictorMono-BoldOblique.woff2
    ├── VictorMono-Bold.woff2
    ├── VictorMono-ExtraLightItalic.woff2
    ├── VictorMono-ExtraLightOblique.woff2
    ├── VictorMono-ExtraLight.woff2
    ├── VictorMono-Italic.woff2
    ├── VictorMono-LightItalic.woff2
    ├── VictorMono-LightOblique.woff2
    ├── VictorMono-Light.woff2
    ├── VictorMono-MediumItalic.woff2
    ├── VictorMono-MediumOblique.woff2
    ├── VictorMono-Medium.woff2
    ├── VictorMono-Oblique.woff2
    ├── VictorMono-Regular.woff2
    ├── VictorMono-SemiBoldItalic.woff2
    ├── VictorMono-SemiBoldOblique.woff2
    ├── VictorMono-SemiBold.woff2
    ├── VictorMono-ThinItalic.woff2
    ├── VictorMono-ThinOblique.woff2
    └── VictorMono-Thin.woff2

9 directories, 148 files
```

Reply:
That's a good point and I don't really understand how the package worked before. I'm new to Nix/NixOS so this might be me not understanding something fully, but here's the issue I encountered:

The package uses `fetchzip`. The `fetchzip` implementation I have found on nixpkgs `master` seems to download the file, unarchive it in `$out` and then delete the archive.
 
The code for version 1.5.2 of this package used `fetchzip` but then unzipped only the files it needed (namely ttf and otf) in the `postFetch` step.

My first attempt was to bump the version number and update the sha256 field but this didn't work because the build script wanted to unzip the archive that `fetchzip` had already deleted.

Now, I'm not sure I did the right thing by modifying `postFetch` but that was the fix I thought was needed for the latest `fetchzip` implementation.

I can delete the extra files from `$out` and update PR (and rebase it on top of master) but it would be really helpful to understand why the old version worked.

Reply:
It might have been that the zip from github contained another zip file, containing the release, and that this changed between version? At least when I try now, both 1.5.2 and 1.5.3 are zipped the same way (only once). Maybe it's a change on github's side? Next option is a change on fetchzip behavior.